stan-ballerina/listener.bal

```diff
@@ -27,49 +27,50 @@ public class Listener {
 
     # Creates a new Streaming Listener.
     #
+    # + url - The NATS Broker URL
     # + streamingConfig - The configuration related to the NATS streaming connectivity
-    public isolated function init(*StreamingConfiguration streamingConfig) returns Error? {
-        self.url = streamingConfig.url;
+    public isolated function init(string url, *StreamingConfiguration streamingConfig) returns Error? {
```

Comment:
string[]|string url ?

Reply:
Fixed in bca5d90d3d0ed31983ec42bff421e142e58cb631